### DISCUSSION TITLE
posts/2023/01/static-mastodon-toots-hugo-tailwind-css-edition/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#49] [Announcements]
# posts/2023/01/static-mastodon-toots-hugo-tailwind-css-edition/

Mastodon is becoming more popular, and Tailwind CSS already is immensely popular, so let’s make them work together in your SSG-based website.

https://www.brycewray.com/posts/2023/01/static-mastodon-toots-hugo-tailwind-css-edition/

**Comment by frankenstein91** [comment]
Can I use the code on my own website or is this just an example?

**Reply by brycewray** [reply]
Yes, the code in all my articles is usable as you wish. My site and its repository are under an MIT license.
